style.css

```diff
@@ -24,7 +24,8 @@
   }
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/217595657-ace9618d-1557-4753-a0db-0b4851dc34bb.png)

**[Optional]**

- [ ] Kindly add the necessary commands for running your project in a local machine in these sub-sections :+1: and if any sub section is unnecessary then kindly remove that sub-section :blush: